pkg/logs/decoder/line_handler.go

```diff
@@ -82,12 +87,22 @@ func (h *SingleLineHandler) process(line []byte) {
 	if lineLen < contentLenLimit {
 		// send content
 		// add 1 to take into account '\n' that we didn't include in content
-		output := NewOutput(content, lineLen+1)
+		parsedMessage, err := h.parser.Parse(content)
+		if err != nil {
+			log.Warn(err)
```

Comment:
would it be worth making `process` return an error ?

Reply:
I don't think so because `process` runs in a goroutine and loops indefinitely
The communication with the tailer is done through a channel, so when there's an error, we don't push a message in the channel (before, the parsing logic was in the tailer and the tailer would skip the message on error)